detector/aws_elasticache_cluster_previous_type.go

```diff
@@ -9,22 +9,21 @@ import (
 
 type AwsElastiCacheClusterPreviousTypeDetector struct {
 	*Detector
-	IssueType         string
-	TargetType        string
-	Target            string
-	DeepCheck         bool
 	previousNodeTypes map[string]bool
 }
 
 func (d *Detector) CreateAwsElastiCacheClusterPreviousTypeDetector() *AwsElastiCacheClusterPreviousTypeDetector {
-	return &AwsElastiCacheClusterPreviousTypeDetector{
+	nd := &AwsElastiCacheClusterPreviousTypeDetector{
 		Detector:          d,
-		IssueType:         issue.WARNING,
-		TargetType:        "resource",
-		Target:            "aws_elasticache_cluster",
-		DeepCheck:         false,
 		previousNodeTypes: map[string]bool{},
 	}
+	nd.Name = "aws_elasticache_cluster_previous_type"
+	nd.IssueType = issue.WARNING
+	nd.TargetType = "resource"
+	nd.Target = "aws_elasticache_cluster"
```

Comment:
__[gometalinter]__
* [goconst] 6 other occurrence(s) of "aws_elasticache_cluster" found in: aws_elasticache_cluster_duplicate_id.go:23:14 aws_elasticache_cluster_invalid_parameter_group.go:23:14 aws_elasticache_cluster_invalid_subnet_group.go:23:14 aws_elasticache_cluster_default_parameter_group.go:20:14 aws_elasticache_cluster_invalid_type.go:23:14 aws_elasticache_cluster_invalid_security_group.go:24:14

<!-- Comment by [SideCI](https://sideci.com) -->